apps/checkout-app/backend/payments/providers/mollie.ts

```diff
@@ -0,0 +1,174 @@
+import createMollieClient, {
+  OrderStatus,
+  OrderLineType,
+  OrderCreateParams,
+} from "@mollie/api-client";
+
+import {
+  OrderFragment,
+  OrderLineFragment,
+  PaymentCreateMutationVariables,
+} from "@/graphql";
+import { APP_URL } from "@/constants";
+
+import { parseAmountToString } from "../utils";
+
+export const mollieClient = createMollieClient({
+  apiKey: process.env.MOLLIE_API_KEY!,
```

Comment:
The key will be stored in the configuration in metadata, it is temporarily obtained from the env variable here, am I right?

Reply:
yep, this is temporary. I'll switch it over to use metadata as a later step as it's a bit easier for development